### PR TITLE
fix(TCOMP-2030) : tomcat version

### PR DESCRIPTION
**Pull request by clesaec**
https://jira.talendforge.org/browse/TCOMP-2030
back port to 1.38 for 1.38.5 version
### Requirements

* Any code change adding any logic **MUST** be tested through a unit test executed with the default build
* Any API addition **MUST** be done with a documentation update if relevant

### CODE REVIEW

File: documentation/src/main/antora/modules/ROOT/pages/_partials/generated_changelog.adoc [hunk 1]
```diff
@@ -45,10 +45,11 @@
 - link:https://jira.talendforge.org/browse/TCOMP-2044[TCOMP-2044^]: Upgrade TSBI to 2.9.2-20211106085418
 
 
+== Version 1.38.4
```
Comment (ypiel-talend) [inline documentation/src/main/antora/modules/ROOT/pages/_partials/generated_changelog.adoc:48]: Is it 1.38.4 or 1.38.5 ?

**Review by ypiel-talend — COMMENTED** [review]
Need to confirm version in changelog .adoc

File: documentation/src/main/antora/modules/ROOT/pages/_partials/generated_changelog.adoc (hunk 1 above)
Comment (clesaec) [reply documentation/src/main/antora/modules/ROOT/pages/_partials/generated_changelog.adoc]: changed

**Review by ypiel-talend — APPROVED** [review]
LGTM

**Pull request merged**